Packs/ServiceNow/Integrations/ServiceNowEventCollector/ServiceNowEventCollector.yml

```diff
@@ -127,8 +127,10 @@ script:
       required: false
       isArray: false
       defaultValue: ""
+  - name: service-now-oauth-login
+    description: Resets the integration context, including stored OAuth refresh token. Use this command when using OAuth 2.0 and encounter access_denied with access token related errors.
```

Comment:
Recommend:
Resets the integration's saved connection information, including the OAuth refresh token. Use this command if you encounter access_denied or other errors related to your access token when using OAuth 2.0.